torch_geometric/sampler/neighbor_sampler.py

```diff
@@ -69,15 +71,23 @@ def __init__(
             self.num_nodes = data.num_nodes
 
             self.node_time: Optional[Tensor] = None
-            if time_attr is not None:
+            self.edge_time: Optional[Tensor] = None
+            if time_attr is not None and temporal_entity == 'node':
```

Comment:
The node time and edge time are mutually exclusive currently -- this is determined by the "temporal_entity" argument.  I have changed it to if... elif. Thanks

Reply:
I was actually wondering why we have separate node_time and edge_time variables when we could just have one "time". Please correct me if I am wrong :)

Reply:
I think it makes sense to separate by `edge_time` and `node_time` on the `pyg-lib` level as otherwise it can be ambigious whether time refers to a node-level or edge-level vector. In the `NeighborSampler` implementation, we don't necessarily need this, since we can infer this from `is_node_attr` or `is_edge_attr`.

Reply:
Thank you for the explanation :)